using_images/other_images/jenkins.adoc

```diff
@@ -600,11 +504,18 @@ an entrypoint. For more details about this, refer to the official
 https://wiki.jenkins-ci.org/display/JENKINS/Distributed+builds#Distributedbuilds-Launchslaveagentheadlessly[Jenkins
 documentation].
 
+<<<<<<< HEAD
```

Comment:
Hey @bmcelvee - I thought I had tagged you on this. There were multiple unresolved conflicts in this cherry pick.

Reply:
Yikes, I'm sorry. The changes (in the original PR) look correct.